Framework/Core/src/WorkflowHelpers.cxx

```diff
@@ -289,6 +291,8 @@ void WorkflowHelpers::injectServiceDevices(WorkflowSpec& workflow, ConfigContext
     std::string prefix = "internal-dpl-";
     if (processor.inputs.empty() && processor.name.compare(0, prefix.size(), prefix) != 0) {
       processor.inputs.push_back(InputSpec{"enumeration", "DPL", "ENUM", static_cast<DataAllocator::SubSpecificationType>(compile_time_hash(processor.name.c_str())), Lifetime::Enumeration});
+      processor.options.push_back(ConfigParamSpec{"orbit-offset-enumeration", VariantType::Int64, 0ll, {"initial value for the enumeration"}});
+      processor.options.push_back(ConfigParamSpec{"orbit-multiplier-enumeration", VariantType::Int64, 1ll, {"initial value for the enumeration"}});
```

Comment:
@ktf I've tried with lines below in the `its-reco-workflow.cxx`
````
auto wf = o2::its::reco_workflow::getWorkflow(...);
for (auto& spec : wf) {
   if (spec.inputs.empty()) {
      LOG(INFO) << "Configuring 1st orbit of " << spec.name;
      spec.options.push_back(ConfigParamSpec{"orbit-offset-enumeration", VariantType::Int64, orbitFirst , {"1st injected orbit"}});
      spec.options.push_back(ConfigParamSpec{"orbit-multiplier-enumeration", VariantType::Int64, nHBFPerTF), {"orbits/TF"}});
   }
}
````
but it leads to 
````
[INFO] Configuring 1st orbit of its-digit-reader
...
[ERROR] error while setting up workflow: 
Throw location unknown (consider using BOOST_THROW_EXCEPTION)
Dynamic exception type: boost::wrapexcept<boost::program_options::ambiguous_option>
std::exception::what: option is ambiguous and matches different versions of 'orbit-multiplier-enumeration'
````

Indeed, in the `o2-its-reco-workflow -h full ` I see duplicated options
````
  --orbit-offset-enumeration arg (=60)         1st injected orbit
  --orbit-multiplier-enumeration arg (=128)  orbits/TF
  --orbit-offset-enumeration arg (=0)            initial value for the enumeration
  --orbit-multiplier-enumeration arg (=1)       initial value for the enumeration
````

Could you add these 2 options after making sure that device has no them already ?
Also, the options hints should be corrected.

Reply:
yes, of course. Looking into it.

Reply:
Should be fixed now. Happy Easter! :-)